src/models/darp6/README.md

```diff
@@ -0,0 +1,46 @@
+# Darter Pro (darp6)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Bonobo WS](./img/darp6.png)
```

Comment:
alt text

Reply:
Corrected with https://github.com/system76/tech-docs/pull/31/commits/9e2fb1fbd50b1492e1c41c07ca2c6b4be8bcf641, thank you.